Automattic/woocommerce-payments
# Add payments activity widget emoji survey

#### Changes proposed in this Pull Request

Emoji-based survey component to capture the user sentiment for the new Payments Overview Widget.

![image](https://github.com/Automattic/woocommerce-payments/assets/1288616/ced1af30-a6a8-4b96-b362-62848ff53729)


#### Todo
- [ ] Place within the Payments Activity Widget introduced in #8412
- [ ] Use common component for close button
- [ ] Check against latest MVP design changes
- [ ] Check/deploy changes to WPCOM survey endpoint
	- [ ] Add WooPayments major version number

#### Testing instructions
- run `wp option update _wcpay_feature_payment_overview_widget 1` in CLI
- apply the patch created in #8314 to your dev server 
- change your host file to point `public-api.wordpress.com` and `dev-mc.a8c.com` to your sandbox's IP address if you didn't already
- log-in to your sandbox
- in the client admin panel, go to `Payments -> Overview`
- by clicking the first 3 emoji, a textarea for comments should appear
- for the highest rates ( last 2 emojis ) the survey will be sent without comments
- after submitting the results, go to https://dev-mc.a8c.com/marketing/survey/index.php?survey=wcpay-payments-overview and check the answers.
- 
https://github.com/Automattic/woocommerce-payments/assets/82826872/71aaf433-8d90-49c9-9e15-a8368aa64707






*

-------------------


**Post merge**
